package.json

```diff
@@ -30,9 +31,9 @@
     "ember-cli-babel": "^5.1.7",
     "ember-cli-eslint": "^3.0.2",
     "ember-cli-version-checker": "^1.3.1",
+    "exists-sync": "0.0.4",
```

Comment:
Aah didn't know that. Will use `fs.existsSync` and update readme for `exists-sync` too.